ah_bootstrap.py

```diff
@@ -92,13 +92,10 @@
         log.error(
             "Error reading setup.cfg: {0!r}\n{1} will not be "
             "automatically bootstrapped and package installation may fail."
-            "\n{2}".format(e, PACKAGE_NAME, _err_help_msg))
```

Comment:
Please do these changes directly in the helpers, ideally `ah_bootstrap.py` should be left alone, just like any other external source code.

Reply:
Reverted.